<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnivariateProblems</class>
 <widget class="QTabWidget" name="UnivariateProblems">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>598</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabBar::tab
{
	 background: qlineargradient(x1: -1, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #4ADFDD,
                                stop: 2 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2.5px solid #CFACFC;
    border-bottom-color: #CFACFC; 
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
    min-width: 10em;
    padding: 2ex;
	font: 87 9pt &quot;Segoe UI Black&quot;;
}

QGroupBox
{
	font: 14pt black &quot;Consolas&quot;;
	
}
QLabel
{
	color: black;
	background-color: rgba(0,0,0,0%);
}
/*QLineEdit#a_input, QLineEdit#b_input,*/
QLineEdit
{
    /*selection-background-color: rgb(233, 99, 0);*/
	background: white;
	color: black;
}</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="BisectionMethod" name="bisectionMethod">
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <attribute name="title">
    <string>Bisection Method</string>
   </attribute>
   <widget class="QGroupBox" name="inputBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>84</y>
      <width>321</width>
      <height>451</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>321</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>500</height>
     </size>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QPushButton" name="button_solve">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>Solve</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="iter_lineEdit">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>130</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>21</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_iterations">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>200</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Set Max iterations (default 10000):</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tol_input">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>21</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="tolerance_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Set tolerance (default .0001):</string>
     </property>
     <property name="indent">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="a_b_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>48</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>48</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>(a, b ) =</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="b_input">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>21</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="ab_comma">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>21</width>
       <height>21</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel::ab_comma
{
	
}</string>
     </property>
     <property name="text">
      <string>,</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="indent">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="a_input">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>21</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="f_x_label">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>34</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>41</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>41</width>
       <height>21</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; color:#000000;&quot;&gt;f (x) =&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="f_x_input">
     <property name="geometry">
      <rect>
       <x>58</x>
       <y>34</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>221</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>221</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="outputBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>321</width>
      <height>451</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>321</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>500</height>
     </size>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QLabel" name="totalSolutions_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string># of solutions (roots)  found</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="totalRoots_lineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>21</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="status_label">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>34</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>41</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>41</width>
       <height>16</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="status_lineEdit">
     <property name="geometry">
      <rect>
       <x>58</x>
       <y>34</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>21</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="flops_lineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>21</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="flops_label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>61</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>61</width>
       <height>16</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>total flops</string>
     </property>
    </widget>
    <widget class="QLabel" name="solution_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;root (solution):&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="solution_lineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>21</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="combox_SPRoutines">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>32</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>181</width>
      <height>22</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_special">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>17</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>17</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Special Purpose Routines</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_general">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>60</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>171</width>
      <height>17</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>171</width>
      <height>17</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>General Purpose Routine</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="fixedMethod">
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <attribute name="title">
    <string>Tab 2</string>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BisectionMethod</class>
   <extends>QWidget</extends>
   <header>bisectionmethod.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
